plugins/out_stackdriver/stackdriver_helper.h

```diff
@@ -0,0 +1,37 @@
+/*  Fluent Bit
+ *  ==========
+ *  Copyright (C) 2019-2020 The Fluent Bit Authors
+ *  Copyright (C) 2015-2018 Treasure Data Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+#ifndef FLB_STD_HELPER_H
+#define FLB_STD_HELPER_H
+
+#include "stackdriver.h"
+
+/* Compare key->via.str and str. Return FLB_TRUE if matches */
+int cmp_str_helper(msgpack_object key, const char* str, const int size);
```

Comment:
maybe better name would help. usually strncmp returns -1/0/1, but in this case it returns FLB_TRUE/FLB_FALSE, so the name and the returned value is a bit confused to me. maybe change the name?

`helper` is not needed by the way - as it is definitely a helper function.

Reply:
Done